tests/pastefromoffice.js

```diff
@@ -3,87 +3,116 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
+import PasteFromOffice from '../src/pastefromoffice';
 import Clipboard from '@ckeditor/ckeditor5-clipboard/src/clipboard';
 import VirtualTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor';
-import DocumentFragment from '@ckeditor/ckeditor5-engine/src/view/documentfragment';
-import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
-
-import PasteFromOffice from '../src/pastefromoffice';
+import HtmlDataProcessor from '@ckeditor/ckeditor5-engine/src/dataprocessor/htmldataprocessor';
 import { createDataTransfer } from './_utils/utils';
+import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
 
-describe( 'Paste from Office plugin', () => {
-	let editor, content, normalizeSpy;
+describe( 'PasteFromOffice', () => {
+	let editor, pasteFromOffice, clipboard;
 
 	testUtils.createSinonSandbox();
 
-	before( () => {
-		content = new DocumentFragment();
-	} );
-
 	beforeEach( () => {
-		return VirtualTestEditor
-			.create( {
-				plugins: [ Clipboard, PasteFromOffice ]
-			} )
-			.then( newEditor => {
-				editor = newEditor;
-				normalizeSpy = testUtils.sinon.spy( editor.plugins.get( 'PasteFromOffice' ), '_normalizeWordInput' );
+		return VirtualTestEditor.create( {
+			plugins: [ PasteFromOffice ]
+		} )
+			.then( _editor => {
+				editor = _editor;
+				pasteFromOffice = editor.plugins.get( 'PasteFromOffice' );
+				clipboard = editor.plugins.get( 'Clipboard' );
 			} );
 	} );
 
-	it( 'runs normalizations if Word meta tag detected #1', () => {
-		const dataTransfer = createDataTransfer( {
-			'text/html': '<meta name=Generator content="Microsoft Word 15">'
-		} );
-
-		editor.plugins.get( 'Clipboard' ).fire( 'inputTransformation', { content, dataTransfer } );
-
-		expect( normalizeSpy.calledOnce ).to.true;
+	it( 'is Paste from Office', () => {
```

Comment:
```suggestion
	it( 'should be loaded', () => {
```
it's named as such as in other plugins